wls-gui-admintool/tests/composables/common/rules.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { beforeEach, describe, expect, it } from "vitest";
+
+import useRules from "@/composables/common/rules.ts";
+
+describe("rules.ts", () => {
+  describe("useRules", () => {
+    let requiredTextRuleBuilder: (
+      requiredText: string
+    ) => (value: unknown) => boolean;
+
+    const requiredText = "text";
+
+    beforeEach(() => {
+      requiredTextRuleBuilder = useRules().requiredText;
+    });
+
+    describe("requiredText", () => {
+      it("should_returnFalse_whenValueIsUndefinedButRequiredTextIsNot", () => {
```

Comment:
```suggestion
      it("should_returnFalse_when_valueIsUndefinedButRequiredTextIsNot", () => {
```
bitte im ganzen file anpassen